feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.ts

```diff
@@ -42,10 +42,6 @@ export class ConfiguratorAttributeDropDownComponent
     this.attributeDropDownForm.setValue(this.attribute?.selectedSingleValue);
   }
 
-  /**
-   * @deprecated since 3.3
-   * Better use onSelect(this.attributeDropDownForm.value)
-   */
   onSelect(): void {
     super.onSelect(this.attributeDropDownForm?.value);
   }
```

Comment:
Shall we remove this method as the docs say?

> ### ConfiguratorAttributeDropDownComponent
> 
> Method `onSelect` has been removed, it is no longer used. Use `onSelect` from super class which takes the > value as argument

Reply:
Good point! I will take care of it

Reply:
Done!